apps/base-docs/docusaurus.config.js

```diff
@@ -108,45 +105,74 @@ const config = {
           ],
         },
         {
-          type: 'doc',
-          docId: 'security',
-          position: 'left',
-          label: 'Security',
+          type: 'dropdown',
+          label: 'Ecosystem',
+          navPosition: 'topRight',
+          items: [
+            {
+              label: 'Apps',
+              href: 'https://www.base.org/ecosystem',
+            },
+            {
+              label: 'Grants',
+              href: 'https://paragraph.xyz/@grants.base.eth/calling-based-builders',
+            },
+          ],
+        },
+        {
+          label: 'Bridge',
+          navPosition: 'topRight',
+          href: 'https://bridge.base.org/',
         },
         {
           type: 'dropdown',
-          label: 'Community',
-          position: 'left',
+          label: 'Developers',
```

Comment:
As part of the design, we discussed updating this to `Builders` instead of `Developers`.